src/Dialect/ONNX/ONNXOps.td.inc

```diff
@@ -5114,6 +5111,7 @@ def ONNXScatterNDOp:ONNX_Op<"ScatterND",
       return {20};
     }
   }];
+  let verifier = [{ return ::verify(*this); }];
```

Comment:
It looks like you are using a different onnx version than the one in `third_party` directory? Normally, this file should be changed with this line only, but not the other lines. Please make sure you are using onnx in `third_party` and regenerate this file.

Reply:
Yeah, I also felt a little confused. My local environment has installed onnx whl before, Is there an impact on it?

Reply:
The onnx installed in my local environment is 1.6.0, the onnx in third_party is 1.9.0. Right ?

Reply:
I see. Yes, I think so. We use onnx to generate operation specifications, so it heavily depends on the onnx version.